CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Changed
 - Validate max. 1000 commands are added to `campaign()`, `events()`, `setupItemProperties()` and `deleteItemProperties()` requests (in accordance with Matej batch API limit).
+- `Lmc\Matej\Model\Response` now contains `getCommandResponse(int $index)` to access individual responses
```

Comment:
Please could you dots at each EOL (for consistency)?

Reply:
Also maybe no need to write FQCN... "`Response` now contains" ... is IMHO sufficiently unambiguous. Similarly for following lines.